<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>upd_rental</class>
 <widget class="QWidget" name="upd_rental">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(89, 108, 134);</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>64</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>126</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>39</y>
     <width>178</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите имя пользователя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>102</y>
     <width>107</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите фильм</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>225</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите время окончания аренды</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>97</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить аренду</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>225</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите время выдачи аренды</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Измените статус</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Активна</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Завершена</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
